region/brinstar/green.json

```diff
@@ -3778,6 +3938,22 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Kihunter Farm",
+                  "notable": false,
+                  "requires": [
+                    {"resetRoom": {
```

Comment:
I think I would like a beam/screw for this farm.

Reply:
Done. Should this also be added to the Crateria Kihunter farm?

Reply:
I'm more ok with that one.  A bit easier to fight them, and crabs are there for a better health farm.
However crabs are on the line of tedious and worth with just power beam.